cfgov/v1/models/base.py

```diff
@@ -67,7 +67,7 @@ class CFGOVPage(Page):
     tags = ClusterTaggableManager(through=CFGOVTaggedPages, blank=True,
                                   related_name='tagged_pages')
     language = models.CharField(
-        choices=ref.supported_languagues, default='en', max_length=2
+        choices=ref.supported_languagues, default='en', max_length=7
```

Comment:
Given the [possible options for language subtags](https://en.wikipedia.org/wiki/IETF_language_tag#Syntax_of_language_tags), and the fact that this field is populated by the set list of choices, I'd propose making this large enough that we don't have to add a migration when we might add a particular language with possible other subtags later.

Reply:
Have we settled on 100?

Reply:
Yup. As @chosak suggests, [Wagtail's built-in `Locale` model uses 100](https://github.com/wagtail/wagtail/blob/b8a86ac15de331654024f3b4590088f0b88626fc/wagtail/core/models.py#L341-L402).

Reply:
Done! https://github.com/cfpb/consumerfinance.gov/pull/6475/commits/b2b97bfd6a5cdf4975a156cc48b5f959a69344fd